src/app/public/modules/split-view/split-view.component.ts

```diff
@@ -74,13 +78,21 @@ import {
 })
 export class SkySplitViewComponent implements OnInit, OnDestroy {
 
+  /**
```

Comment:
This is missing the default value. The current docs show the default value as `Back to list`.